### PR TITLE
docs: improve main readme

### PR DESCRIPTION
**Pull request by Nikeleos**
- some yarn commands
- browsers versions supported

Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [x] Testing
- [x] No error logs in web console (except iFrame errors)
- [x] Create or Update documentation (README file for users/integrators)
- [x] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature 
- [x] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: doc/nonFunctionalRequirements.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Non functional requirements
+- There is currently no connexion timeout
```
Comment (csm-thu) [inline doc/nonFunctionalRequirements.md:2]: Typo: connection

File: README.md [hunk 2]
```diff
@@ -66,7 +67,7 @@ need to use this command every time.
 ## Available Scripts
 
 ### React scripts
-This application has been created with *create-react-app*, that provides some scripts directly in the project directory.
+This application has been created with *create-react-app*, that provides some scripts directly in the project directory.\
```
Comment (csm-thu) [inline README.md:70]: Remove the trailing backslash ;)

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 3]
```diff
@@ -91,6 +92,14 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
+#### `yarn cache clean --force`
+
+Clear yarn cache.
+
+#### `yarn cache clean --force && rm -rf node_modules && yarn install`
+
+In order to be sure to correctly "clean" the app, this script is prefered from the one above.
```
Comment (FMO-Cosmo) [inline README.md:101]: I don't understand here what script is preferred to which one. Plus, it seemed to me that clearing the cache was really important but can we add a little sentence to explain in what case(s) it needs to be done ?

Comment (Nikeleos) [reply README.md:101]: Just clean the cache may not always be suffcient to "clean" the webapp after package changes.
Sometimes, it also requires node_module directory remove then yarn install.

**Pull request merged**